python/dataproc_templates/hive/util/README.md

```diff
@@ -0,0 +1,54 @@
+# Hive Spark DDL Extractor To BigQuery
+
+Template for extracting DDL data from Hive and writing to BigQuery table.
+
+It uses [Spark SQL & Hive Integration](https://cloud.google.com/architecture/using-apache-hive-on-cloud-dataproc#querying_hive_with_sparksql) for querying Hive with SparkSQL.
+
+## Arguments
+
+* `spark.hadoop.hive.metastore.uris`: Hive metastore URI
+* `hivesparkddl.bigquery.input.database`: Hive database for input table
+* `hivesparkddl.bigquery.output.dataset`: BigQuery dataset for the output table
+* `hivesparkddl.bigquery.output.table`: BigQuery output table name
+* `hivesparkddl.bigquery.output.bucket`: Output bucket
+
+## Usage
+
+```
+$ python main.py --template HIVESPARKDDLTOBIGQUERY --help
```

Comment:
I think template name should be HIVESPARKDDLTOBQ as mentioned in main.py. Please make required changes everywhere